rules/eslint/best-practices/on.js

```diff
@@ -1,27 +1,27 @@
 "use strict";
 
 module.exports = {
-  "rules": {
+  rules: {
     // Enforces getter/setter pairs in objects
     "accessor-pairs": 0,
     // enforce return statements in callbacks of array’s methods
     "array-callback-return": 0,
     // treat var statements as if they were block scoped
     "block-scoped-var": 0,
     // specify the maximum cyclomatic complexity allowed in a program
-    "complexity": [2, 11],
+    complexity: [2, 11],
     // require return statements to either always or never specify values
     "consistent-return": 2,
     // specify curly brace conventions for all control statements
```

Comment:
@Nehle Not sure what you mean - the new rule specifies that object properties should not have quotes unless needed (such as if they contain a `-`).